### PR TITLE
#166869465 Update swagger documentation

### PR DESCRIPTION
**Pull request by cdvx**
**What does this PR do?**
* update swagger documentation

**Description of Task to be completed?**
* update swagger documentation

**How should this be manually tested?**
* Pull and checkout to this branch locally by running `git fetch origin ch-update-swagger-documentation-166869465 && ch-update-swagger-documentation-166869465`
* Start  the virtual environment `source venv/bin/activate`
* run tests `pytest`

**Any background context you want to provide?**
* N/A

**What are the relevant pivotal tracker stories?**
* [166869465](https://www.pivotaltracker.com/story/show/166869465)

### CODE REVIEW

File: redis.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ ! -d redis-stable/src ]; then
+  curl -O http://download.redis.io/redis-stable.tar.gz
+  tar xvzf redis-stable.tar.gz
+  rm redis-stable.tar.gz
+fi
+cd redis-stable
+make
+src/redis-server
```
Comment (kimbugp) [inline redis.sh]: @cdvx, could you also add documentation in the readme of how to use this.

Comment (kimbugp) [reply redis.sh]: I also think this can be in a different commit for tracking

Comment (cdvx) [reply redis.sh]: Thanks for that insight @kimbugp, let me add documentation to the README.md on how to use it

**Comment by kimbugp** [comment]
@cdvx LGTM

**Pull request merged**